### DISCUSSION TITLE
Can't open "corrupted" file

### DISCUSSION THREAD

**Discussion by IoanTeofil** [xournalpp/xournalpp#2828] [Q&A] [answered]
Hello! I have a problem with this app.
So I wrote something in it. I saved the file and now I can't open it anymore:    Error opening file: "..."
I might have pressed "Annotate PDF" (if it matters. I saw something related to this in other problems).
In properties, the file has 2MB+. Do you have any solution?
Thanks in advance!

**Comment by rolandlo** [comment, marked as answer]
If you upload the file I can take a look at it.

**Reply by IoanTeofil** [reply]
Can you give me your email address?

**Reply by rolandlo** [reply]
I would prefer not to. You can contact me on the [Gitter channel](https://gitter.im/xournalpp/xournalpp).

**Reply by rolandlo** [reply]
Note: The file was a  corrupted .xoj-file created in Xournal (not Xournal++), which consisted entirely of 0's. So there was no information to recover.
